com.github.CineEncoder.yaml

```diff
@@ -0,0 +1,17 @@
+app-id: com.github.CineEncoder
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: cine-encoder
```

Comment:
You need to specify finish-args with the set of permissions your application requires in order for it to run.